index.js

```diff
@@ -98,7 +98,7 @@ function filter (obj, { protoAction = 'error', constructorAction = 'error', safe
 }
 
 function parse (text, reviver, options) {
-  const stackTraceLimit = Error.stackTraceLimit
+  const { stackTraceLimit } = Error
```

Comment:
For this I want to see benchmarks proving we dont have perf degredations.

Reply:
@Uzlopak 

Before:

![image](https://github.com/fastify/secure-json-parse/assets/43814140/9a6c4a45-0888-46cf-aa1a-01e7f83dcc2a)

After:

![image](https://github.com/fastify/secure-json-parse/assets/43814140/40729bc0-b26c-4cdc-abc4-9ef1ba3efa67)